homeassistant/components/nederlandse_spoorwegen/utils.py

```diff
@@ -0,0 +1,140 @@
+"""Utility functions for Nederlandse Spoorwegen integration."""
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Any
+
+from ns_api import Trip
+
+
+def get_planned_departure_platform(trip: Trip) -> str | None:
+    """Get planned departure platform from trip data."""
+    return trip.departure_platform_planned
```

Comment:
This is a proxy method, and only add unneeded method calls and makes code harder to follow (another function jump needs to be processed in ones brain).

I would strongly advise against that and review the same for all of the below methods.